### ISSUE TITLE
Renaming fluid ports

### ISSUE THREAD

**Issue by Mathadon** [open-ideas/IDEAS#586]
The two zone fluid ports are currently called `flowPort_In` and `flowPort_Out`. These names are from the time when we used  `FlowPort` instead of `FluidPort`. To be in line with typical naming in IDEAS and annex60 I'd like to rename these ports to `port_a` and `port_b` in IDEAS 1.0.

This would break your models of course, but I can try to write a conversion script to ease the pain a bit.. @rubenbaetens @damienpicard @GlennReynders @cprotopa @bramvdh91 is this ok for you?

**Comment by rubenbaetens** [comment]
Theoretically it should then extend from `TwoPort` ...

**Comment by Mathadon** [comment]
It could if it weren't for the fact that the ports are conditional :)

**Comment by damienpicard** [comment]
I wrote a python script to change strings in subfolder. The only draw back is that it changes the line ending... (I think).
Another option is to use the search tool of Dymola on `modelica text`.

Any way, the script is:
```
"""
This file search in a whole folder and replace a string by another one.
"""
import glob
import os
from fnmatch import fnmatch

def listFilesInSubDir(root, pattern, printFiles = False):
    """
    Return all files containing the pattern in the root directory
    :param root:
    :param pattern:
    :param printFiles:
    :return:
    """
    filesList = []
    for path, subdirs, files in os.walk(root):
        for name in files:
            for p in pattern:
                if fnmatch(name, p):
                    filesList.append(os.path.join(path, name))
                    if printFiles:
                        print filesList[-1]
    return filesList

def containString(filesList, string):
    """
    Print all file names from  filesList with contains string
    :param filesList:
    :param string:
    :return:
    """
    for f in filesList:
        for s in string:
            if s in open(f).read():
                print f

def replaceLine(filesList, replacements):
    """
    Replace keys by arguments from replacements for all files in filesList
    :param filesList:
    :param replacements:
    :return:
    """
    for f in filesList:
        lines = []
        with open(f) as infile:
            for line in infile:
                for src, target in replacements.iteritems():
                    line = line.replace(src, target)
                lines.append(line)
        with open(f, 'w') as outfile:
            for line in lines:
                outfile.write(line)

if __name__ == '__main__':
    root = 'C:/Users/u0085306/Documents/work/modelica/SmartGeotherm'
    pattern = ["*.mo"]
    rep = {'hsPar' : 'heaSysPar','venPar' : 'venSysPar'}

    containString(listFilesInSubDir(root, pattern, printFiles = False),rep.keys())

    #listFilesInSubDir(root, pattern, printFiles = True)
    replaceLine(listFilesInSubDir(root, pattern, printFiles = False), rep)








```

**Comment by Mathadon** [comment]
following script seems to work:

```
clear

convertClear();

convertElement("IDEAS.Buildings.Components.Zone", "flowPort_Out", "port_b");
convertElement("IDEAS.Buildings.Components.Zone", "flowPort_In", "port_a");
```

**Comment by Mathadon** [comment]
I added 

```
convertModifiers({"IDEAS.Fluid.FixedResistances.FixedResistanceDpM"},
                  {"dh"},
                  fill("", 0),
                  true);

convertModifiers({"IDEAS.Fluid.FixedResistances.FixedResistanceDpM"},
                  {"ReC"},
                  fill("", 0),
                  true);

convertModifiers({"IDEAS.Fluid.FixedResistances.FixedResistanceDpM"},
                  {"use_dh"},
                  fill("", 0),
                  true);

convertClass("IDEAS.Fluid.FixedResistances.FixedResistanceDpM",
             "IDEAS.Fluid.FixedResistances.PressureDrop");

// Flow junction
convertModifiers({"IDEAS.Fluid.FixedResistances.SplitterFixedResistanceDpM"},
                  {"dh"},
                  fill("", 0),
                  true);

convertModifiers({"IDEAS.Fluid.FixedResistances.SplitterFixedResistanceDpM"},
                  {"ReC"},
                  fill("", 0),
                  true);

convertModifiers({"IDEAS.Fluid.FixedResistances.SplitterFixedResistanceDpM"},
                  {"use_dh"},
                  fill("", 0),
                  true);

convertClass("IDEAS.Fluid.FixedResistances.SplitterFixedResistanceDpM",
             "IDEAS.Fluid.FixedResistances.Junction");
```

for dealing with some changes in Annex 60.

**Comment by Mathadon** [comment]
closed in #619
